routes/patient/PatientSymptomEntrys.js

```diff
@@ -0,0 +1,58 @@
+const router = require('express').Router();
+let PatientSymptomEntry = require('../../models/PatientSymptomEntry');
+
+router.route('/patientsymptomentries').get((req, res) => {
+
+      PatientSymptomEntry.find()
+        .then(patientsymptomentries => res.json(patientsymptomentries))
+        .catch(err => res.status(400).json('Error: ' + err));
+});
+
+
+router.route('/PatientSymptomEntrys/add').post((req, res) => {
```

Comment:
This route looks good! In the create-symptom component on front end is directed to '/patients/patientEntry/add'.  If you change that to this route it might be a more favorable result.